src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,23 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@each $breakpoint, $columns in $breakpoints {
+  @media (min-width: #{$breakpoint}) {
+    .catalog {
+      grid-template-columns: repeat(#{$columns}, 200px);
+    }
+  }
+}
+
+.card:hover {
+  transform: scale(1.2);
+}
+
+.card:hover .card__title {
+  color: $hover-text-color;
+}
```

Comment:
Move the to the card as they are not related to the catalog block, and use nesting as well